plugins/languages/otter.nix

```diff
@@ -83,4 +83,12 @@ helpers.neovim-plugin.mkNeovimPlugin config {
       When true, otter handles these cases fully. This is a (minor) performance hit.
     '';
   };
+
+  extraConfig = cfg: {
+    warnings = lib.optional (!(cfg.enable -> config.plugins.treesitter.settings.highlight.enable)) ''
```

Comment:
```
error: expected a Boolean but found null: null
at /nix/store/l6ml4gkff7nzqfqwz4j86a696cnfpcql-source/plugins/languages/otter.nix:88:43:
87| extraConfig = cfg: {
88| warnings = lib.optional (!(cfg.enable -> config.plugins.treesitter.settings.highlight.enable)) ''
| ^
```

Since the `highlight.enable` settings option is nullable, we need to handle that.

Note: if you did `enable == true` then `nix fmt` would have "fixed" that for you, leading to this error.

These would be true when enable is null: `enable != null -> enable` or `isBool enable -> enable`

But I think we actually want false when null, so lets go with: `enable != null && enable` or `isBool enable && enable`.

Probably best to have a small helper in a let block: `isTrue = v: isBool v && v`, or maybe we can use [these helpers](https://github.com/nix-community/nixvim/blob/216d64c158da5523d5b3db0895e1345175c21502/lib/utils.nix#L106-L110)?

Reply:
Could we only check for `null` and not `false`? Maybe the user will do _something else_ to enable it, and as such we don't want to warn him each time?